automated install/basic-install.sh

```diff
@@ -1434,7 +1437,7 @@ stop_service() {
 }
 
 # Start/Restart service passed in as argument
-start_service() {
+restart_service() {
     # Local, named variables
     local str="Starting ${1} service"
```

Comment:
Perhaps change this to : 
```suggestion
    local str="Restarting ${1} service"
```

Reply:
Other than that, Looks good to me.